service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/Party.java

```diff
@@ -10,11 +11,14 @@
 
 @Data
 @AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class Party {
     public final String partyId;
     public final PartyType partyType;
+    public final String title;
```

Comment:
please remove

Reply:
When i try and remove it complains and still asks for them ?

Reply:
@rebeccabrennan what file complains?

Reply:
👍